docs/Getting Started/Install and Run/index.md

```diff
@@ -0,0 +1,96 @@
+# Installation
+A quick guide to installing, configuring, and running an instance of the SDL Core on a linux OS (default environment is Ubuntu 18.04 LTS).
+
+First, clone [SDL Core](https://github.com/smartdevicelink/sdl_core), then create a folder for your build outside of the source folder.
```

Comment:
With the migration of jsoncpp to a submodule, and the impending addition of the rpc_generator submodule, we should leave a note near here that you will also need to acquire the submodules when cloning.

This can be done either with the `--recurse-submodules` flag on core, or by running the following two commands after cloning:
```
git submodule init
git submodule update
```